modules/default/calendar/calendar.js

```diff
@@ -72,10 +72,18 @@ Module.register("calendar", {
 
 			var calendarConfig = {
 				maximumEntries: calendar.maximumEntries,
-				maximumNumberOfDays: calendar.maximumNumberOfDays,
+				maximumNumberOfDays: calendar.maximumNumberOfDays
 			};
 
-			this.addCalendar(calendar.url, calendar.user, calendar.pass, calendarConfig);
+			// we check user and password here for backwards compatibility with old configs
+			if(calendar.user && calendar.pass){
```

Comment:
I don't really understand this. How and where should deprecation options appear?